test/utils/ShortStrings.t.sol

```diff
@@ -25,7 +25,7 @@ contract ShortStringsTest is Test {
 
     function testRevertLong(string memory input) external {
         vm.assume(!_isShort(input));
-        vm.expectRevert(abi.encodeWithSelector(ShortStrings.StringTooLong.selector, input));
+        vm.expectRevert(abi.encodeCall(ShortStrings.StringTooLong, (input)));
```

Comment:
```suggestion
        vm.expectRevert(abi.encodeWithSelector(ShortStrings.StringTooLong.selector, input));
```

Reply:
Thank You for pointing this out @Amxx, however, when I ran `npm run test` all tests were passing.
I don't see anything related to running foundry tests, thus, missed this one.
Committing this suggestion and resolving it.
Thanks again.